packages/cozy-harvest-lib/src/components/Routes.jsx

```diff
@@ -85,6 +85,8 @@ const Routes = ({ konnectorRoot, konnector, onDismiss, datacardOptions }) => {
                             accountId={match.params.accountId}
                             accountsAndTriggers={accountsAndTriggers}
                             onDismiss={onDismiss}
+                            showNewAccountButton={!konnector.clientSide}
+                            showAccountSelection={!konnector.clientSide}
```

Comment:
nit: please add 1 test to test this feature/bug (not mandatory, as Routes.jsx is not tested and looks not so easy to test.)

Reply:
sorry I will pass this time since it is indeed not easy to test an this test does not have a lot of value